presto-resource-group-managers/src/main/java/com/facebook/presto/resourceGroups/db/ResourceGroupsDao.java

```diff
@@ -79,6 +79,7 @@
             "  query_type VARCHAR(512),\n" +
             "  client_tags VARCHAR(512),\n" +
             "  selector_resource_estimate VARCHAR(1024),\n" +
+            "  client_info_regex VARCHAR(512),\n" +
```

Comment:
Let's make this larger, since client infos are more complicated than tags or query types.

Reply:
Makes sense, I changed the client info regex size to 1024